Sources/Moya/Response.swift

```diff
@@ -142,3 +158,7 @@ public extension Response {
         }
     }
 }
+
+private struct DecodableWrapper<T: Decodable>: Decodable {
```

Comment:
Can you explain why we had to add this wrapper?

Reply:
Because the base value types cannot serialize to json data, so I wrap the value to a Dictionary, then serialize the Dictionary to Data. So when decode the data, the result should be unwrapped.